### DISCUSSION TITLE
github.com/hbstack/blog/v0.25.0

### DISCUSSION THREAD

**Discussion by giscus** [hbstack/site#761] [General]
# github.com/hbstack/blog/v0.25.0

0.25.0 (2024-02-11) Features ✨ add the toc page parameter, when false then hide the ToC (#646) (c791065) 

https://hbstack.dev/releases/github.com-hbstack-blog/v0.25.0/

**Comment by pleasantone** [comment]
do you want to document it (and the other fields) in decap-cms and archetypes/blog/index.md?

**Reply by razonyang** [reply]
I am currently on vacation, PR are welcome.

**Comment by pleasantone** [comment]
Have a fantastic vacation and happy new year!!!

**Reply by razonyang** [reply]
Thanks, the field is available since [github.com/hbstack/decap-cms@v0.3.0](https://github.com/hbstack/decap-cms/releases/tag/v0.3.0), I don't put it into blog/modules/decap-cms module to keep the `hb_blog_posts` config as simple as possible to include required/common parameters only.

For other fields, `_extends` should work, for example.

```
collections:
  foobar:
    _extends:
      - hb_blog_posts
      - hb_field_toc
```
